<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>sans-serif</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
background: transparent;
border: none;
border-bottom: 1px solid rgba(0, 0, 0, 155);
font-family: sans-serif;
}

QCheckBox::indicator{
border-radius: 5px;
border: 2px solid rgb(170, 204, 206);
width : 10px;
height : 10px;
}
QCheckBox::indicator:checked{
border-radius: 5px;
background: rgb(52, 161, 185);
border: 2px solid rgb(170, 204, 206);
width : 10px;
height : 10px;
}


QMainWindow{
background: rgb(0, 90, 106);
font-family: sans-serif;
}

QGroupBox{
background: white;
border: 1px solid rgba(255, 255, 255, 200);
font-family: sans-serif;
border-radius:5px;
}
QGroupBox#statusBox{
border-radius: 0px;
}
QPushButton#exitButton, QPushButton#minimizeButton{
border-radius: 0px;
border: none;
background: transparent;
}
QPushButton#exitButton:hover{
background: rgb(255, 46, 46);
}
QPushButton#minimizeButton:hover{
background: rgb(223, 223, 223);
}
QPushButton:hover{
background: rgb(75, 195, 220);
border: none;
}

QPushButton{
background: rgb(58, 180, 208);
border-radius: 10px;
border: 1px solid rgb(18, 140, 168);
font-family: sans-serif;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>225</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="regButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вход</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Вход</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="stayCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Оставаться в системе</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>—</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Glow</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
